examples/calc_flatfield.py

```diff
@@ -0,0 +1,112 @@
+"""
+Extract flat field coefficients from flasher data files.
```

Comment:
I believe it would be good to mention in this doc string here, how this example should be executed. 
Don't hesitate to be very specific in an example. E.g. mention real and exact file-names. Even if a possible user, might not have that file. They still gained some knowledge here, namely which file they need to ask for.